### ISSUE TITLE
Possible to release projector weights too?

### ISSUE THREAD

**Issue by lolemacs** [facebookresearch/barlowtwins#15]
Hi,

Thanks for releasing the pre-trained weights for the ResNet-50 backbone! Would it be possible to also release the weights of the pre-trained projector associated with the pre-trained ResNet-50?

This would be extra helpful for downstream tasks where there is considerably more unlabeled than labeled data, making it easy to fine-tune the whole model (backbone encoder + projector) by minimizing the BT loss on the unlabeled samples prior to training a classifier on the labeled subset.

**Comment by jzbontar** [comment]
Yes, absolutely. I've update the README to include the link to the full checkpoint.
